Fix the example

kasuken opened the pull request:
If you copy and paste the code "as is", obtains this error: https://stackoverflow.com/questions/12308440/azure-service-bus-brokeredmessage-has-already-been-consumed

azurecla commented:
Hi __@deneha__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented:
The pull request shows more diffs than I actually did.  I just changed an issue with how Step 4c was rendering.

Commit:
Fix the example

If you copy and paste the code "as is", obtains this error: https://stackoverflow.com/questions/12308440/azure-service-bus-brokeredmessage-has-already-been-consumed
